### DISCUSSION TITLE
Обход X / Twitter

### DISCUSSION THREAD

**Discussion by EvgenijTkachuk** [Flowseal/zapret-discord-youtube#689] [Ideas]
Добавьте, пожалуйста, обход ограничений X. 
X + Discord - это жизненно необходимая связка в крипте. Приходится дополнительно включать VPN расширение, чтобы и к твиттеру доступ был.

**Comment by killedbyasphyxia** [comment]
@EvgenijTkachuk насколько я знаю домены в list-general.txt можно менять, так что просто вставь туда x.com
Вроде должен распространяться и на поддомены, но не уверен так что потести.

**Reply by zerstauber** [reply]
добавил в список ссылку на твиттер, перезапустил компутер и ничего не поменялось, хотя всё по  туториалу

**Reply by poiuhv** [reply]
Попробуй добавить в list-general abs.twimg.com

**Reply by supersledopytus-create** [reply]
Добавил
x.com
twitter.com
Всё заработаоло.

**Reply by SupCupKind** [reply]
А это можно сделать в ByeDPI для Android?

**Reply by ekungurov** [reply]
@SupCupKind можно, а зачем?

**Comment by poiuhv** [comment]
Для обхода блокировок на сайты нужно не только вводить сам адрес сайта, но и доп. адреса которые выделены под медиа контент.
У твиттера это например:
x.com
abs.twimg.com
video.twimg.com
pbs.twimg.com
Чтобы узнать можно в браузере открыть консоль Java и посмотреть ошибки, в них пишут адрес с которого не приходит ответ.
Для примера вот,  можно увидеть что нет доступа к api.x.com и его тоже надо добавлять в list-general
![image](https://github.com/user-attachments/assets/ea4d7fac-ea96-4b6a-883a-7aec00169525)

**Reply by kogkodeniska** [reply]
> Для обхода блокировок на сайты нужно не только вводить сам адрес сайта, но и доп. адреса которые выделены под медиа контент. У твиттера это например: x.com abs.twimg.com video.twimg.com pbs.twimg.com Чтобы узнать можно в браузере открыть консоль Java и посмотреть ошибки, в них пишут адрес с которого не приходит ответ. Для примера вот, можно увидеть что нет доступа к api.x.com и его тоже надо добавлять в list-general ![image](https://private-user-images.githubusercontent.com/78111176/399233166-ea4d7fac-ea96-4b6a-883a-7aec00169525.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ2NDM0OTcsIm5iZiI6MTc0NDY0MzE5NywicGF0aCI6Ii83ODExMTE3Ni8zOTkyMzMxNjYtZWE0ZDdmYWMtZWE5Ni00YjZhLTg4M2EtN2FlYzAwMTY5NTI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDE0VDE1MDYzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzMzEzYWM0OTIxMzJjMjg2MTJjYTU0ZWFlMjM5YjM4M2FjOGRlOWMyYmFmNTM4YTMzOTA1NjdhNGVhZTg3NDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PoxhKiAwZiTaiZhQrdChNmSg0wDZo16Nw43vXL2rDKw)

открываю консоль ошибок сайта. Они отсутствуют, но твиттер не грузит. Вот ссылки, которые закинул в генерал.
![image](https://github.com/user-attachments/assets/08ef9c11-7500-4cab-b249-d07e1807c0bb)

**Reply by kogkodeniska** [reply]
а вот сама консоль
![image](https://github.com/user-attachments/assets/82ea1953-97e4-4174-bd85-3aec8a55d9a4)

**Comment by ivaqis** [comment]
Добавил такие адреса к general и работает отлично, с ads'ами наверное можно и убрать:
api.x.com
twitter.com
api.twitter.com
api.tweetdeck.com
abs.twimg.com
pbs.twimg.com
video.twimg.com
t.co
mobile.twitter.com
support.twitter.com
syndication.twitter.com
abs-0.twimg.com
upload.twitter.com
static.ads-twitter.com

**Reply by sadfov** [reply]
главная страница не загружается, вставил все указанные адреса, но вот такой результат
![изображение](https://github.com/user-attachments/assets/2758abda-104d-469e-893d-c672f2d50da8)
и вот что пишет в консоли
![изображение](https://github.com/user-attachments/assets/cb1cd5c3-7439-4bd9-b9e6-32dacbbe9995)

**Reply by poiuhv** [reply]
Опиши пожалуйста как включаешь сам zapret: какой файл запускаешь, включаешь ли как сервис, запускаешь ли от имени администратора. 
Также возможно трабл есть в том как заносишь адреса в list-general, тоже бы не помешало взглянуть. 
Я например на веру сразу использовал service_install сразу после того как обновлял список адресов.

**Reply by ArNfrSh** [reply]
Такая же трабла была - внес адреса в лист, но не прогружалось ничего. Помог запуск альтернативного батника (ALT2) в моём случае.

**Reply by poiuhv** [reply]
> Такая же трабла была - внес адреса в лист, но не прогружалось ничего. Помог запуск альтернативного батника (ALT2) в моём случае.

Такая же история, использую на постоянке

**Reply by ivaqis** [reply]
> главная страница не загружается, вставил все указанные адреса, но вот такой результат ![изображение](https://private-user-images.githubusercontent.com/181337788/399556027-2758abda-104d-469e-893d-c672f2d50da8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU3Mzk2OTgsIm5iZiI6MTczNTczOTM5OCwicGF0aCI6Ii8xODEzMzc3ODgvMzk5NTU2MDI3LTI3NThhYmRhLTEwNGQtNDY5ZS04OTNkLWM2NzJmMmQ1MGRhOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwMVQxMzQ5NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNTdjMGM0MTMzODBlYThhYTE3OTdmYWZkYzljMzljYTQxM2IwYmRmOTg0ZTBiYmMzMjc1YjU4ZDA3ZjY1NjY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.DWs3fqwuTZNH5sJz64tt2DLhipBi9RXj1TzIF7mAYgc) и вот что пишет в консоли ![изображение](https://private-user-images.githubusercontent.com/181337788/399556050-cb1cd5c3-7439-4bd9-b9e6-32dacbbe9995.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU3Mzk2OTgsIm5iZiI6MTczNTczOTM5OCwicGF0aCI6Ii8xODEzMzc3ODgvMzk5NTU2MDUwLWNiMWNkNWMzLTc0MzktNGJkOS1iOWU2LTMyZGFjYmJlOTk5NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwMVQxMzQ5NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hM2M5NDBiYmZkOTJmZjg4YTFhNmUxMDBlMjg0YTk1YjRiYWQ0MTE4NTEzZjRmNmU3ODZhM2JjZWVkNjhjNTIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.vbDAmB61jic8Z-7KWKMG0ISI2POP5oMahtXpORpzfFM)

У меня в Network в инструментах разработчика все статусы прогрузки апишек твитера с 200 статусом, так что в целом адреса рабочие, бывает редко непрогрузы, а в основном норм грузит, может тут дело уже в чем-то другом

**Reply by sadfov** [reply]
> Опиши пожалуйста как включаешь сам zapret: какой файл запускаешь, включаешь ли как сервис, запускаешь ли от имени администратора. Также возможно трабл есть в том как заносишь адреса в list-general, тоже бы не помешало взглянуть. Я например на веру сразу использовал service_install сразу после того как обновлял список адресов.

general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора.
вот так записаны адреса, как и остальные 
![изображение](https://github.com/user-attachments/assets/fc7e19ca-f34b-49f2-ae19-e4cc70f94b9d)

**Reply by poiuhv** [reply]
> general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора. вот так записаны адреса
Видать какие-то разные у нас релизы скачаны, у меня ни в одной сборке не было батников с названием global.bat 
Если что сам использую zapret-discors-youtube 1.6.1

**Reply by sadfov** [reply]
> > general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора. вот так записаны адреса
> > Видать какие-то разные у нас релизы скачаны, у меня ни в одной сборке не было батников с названием global.bat
> > Если что сам использую zapret-discors-youtube 1.6.1

я перепутал название, там general, тут моя ошибка, айкю как у хлеба(может даже меньше)

**Reply by poiuhv** [reply]
> > > general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора. вот так записаны адреса
> > > Видать какие-то разные у нас релизы скачаны, у меня ни в одной сборке не было батников с названием global.bat
> > > Если что сам использую zapret-discors-youtube 1.6.1
> 
> я перепутал название, там general, тут моя ошибка, айкю как у хлеба(может даже меньше)

А кстати service_install в итоге пробовал? Я через него пользуюсь, чтобы не надо было запускать отдельный батник каждый раз(он вроде как должен быть постоянно включён для работы), ну и на всякий возможно забыл сохранить текстовый док чтобы подтвердить изменение(не думаю что глупый, но всякое бывает, от таких ошибок никто не защищён)

**Reply by sadfov** [reply]
> > > > general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора. вот так записаны адреса
> > > > Видать какие-то разные у нас релизы скачаны, у меня ни в одной сборке не было батников с названием global.bat
> > > > Если что сам использую zapret-discors-youtube 1.6.1
> > 
> > 
> > я перепутал название, там general, тут моя ошибка, айкю как у хлеба(может даже меньше)
> 
> А кстати service_install в итоге пробовал? Я через него пользуюсь, чтобы не надо было запускать отдельный батник каждый раз(он вроде как должен быть постоянно включён для работы), ну и на всякий возможно забыл сохранить текстовый док чтобы подтвердить изменение(не думаю что глупый, но всякое бывает, от таких ошибок никто не защищён)

раньше запускал просто через батник, щяс же сделал через service_install, но это не меняет сути того, что x не загружает главную страницу, меня интересует конкретно это

**Reply by poiuhv** [reply]
> > > > > general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора. вот так записаны адреса
> > > > > Видать какие-то разные у нас релизы скачаны, у меня ни в одной сборке не было батников с названием global.bat
> > > > > Если что сам использую zapret-discors-youtube 1.6.1
> > > 
> > > 
> > > я перепутал название, там general, тут моя ошибка, айкю как у хлеба(может даже меньше)
> > 
> > 
> > А кстати service_install в итоге пробовал? Я через него пользуюсь, чтобы не надо было запускать отдельный батник каждый раз(он вроде как должен быть постоянно включён для работы), ну и на всякий возможно забыл сохранить текстовый док чтобы подтвердить изменение(не думаю что глупый, но всякое бывает, от таких ошибок никто не защищён)
> 
> раньше запускал просто через батник, щяс же сделал через service_install, но это не меняет сути того, что x не загружает главную страницу, меня интересует конкретно это

Из самого такого осталось только наверное что расширения в браузере могут повлиять, сам браузер, или мб настройка по типу куки файлов хз. Погуглив по твоим ошибкам из вариантов только кэш почистить, ну и для чисто эксперимента без расширений и в другом браузере попытаться стартануть

**Reply by sadfov** [reply]
> > > > > > general.bat у меня не робит, я перетыкал несколько вариков и нашёл alt3, на нём работает и дс и ютуб. Запускаю от имени администратора. вот так записаны адреса
> > > > > > Видать какие-то разные у нас релизы скачаны, у меня ни в одной сборке не было батников с названием global.bat
> > > > > > Если что сам использую zapret-discors-youtube 1.6.1
> > > > 
> > > > 
> > > > я перепутал название, там general, тут моя ошибка, айкю как у хлеба(может даже меньше)
> > > 
> > > 
> > > А кстати service_install в итоге пробовал? Я через него пользуюсь, чтобы не надо было запускать отдельный батник каждый раз(он вроде как должен быть постоянно включён для работы), ну и на всякий возможно забыл сохранить текстовый док чтобы подтвердить изменение(не думаю что глупый, но всякое бывает, от таких ошибок никто не защищён)
> > 
> > 
> > раньше запускал просто через батник, щяс же сделал через service_install, но это не меняет сути того, что x не загружает главную страницу, меня интересует конкретно это
> 
> Из самого такого осталось только наверное что расширения в браузере могут повлиять, сам браузер, или мб настройка по типу куки файлов хз. Погуглив по твоим ошибкам из вариантов только кэш почистить, ну и для чисто эксперимента без расширений и в другом браузере попытаться стартануть

расширения все вырубил, попробовал ничего, сменил браузер, тоже офф расширения, также не работает, ну да ладно, как нибудь справляюсь, спасибо за уделённое время(и простите мне мою тупость)

**Reply by Scoutx80** [reply]
а как ты нашел все домены и поддомены связанные с сайтом?

**Reply by sadfov** [reply]
> а как ты нашел все домены и поддомены связанные с сайтом?

Через консоль f12 нажми и там поищи.
А так по этому способу слишком геморно сидеть, проще ВПН купить и не париться

**Reply by Scoutx80** [reply]
> > а как ты нашел все домены и поддомены связанные с сайтом?
> 
> Через консоль f12 нажми и там поищи. А так по этому способу слишком геморно сидеть, проще ВПН купить и не париться

Да у меня проблема с впн, IPsec не могу найти открыть IK2v и тд, мне проще будет домены находить для запрета чем впн чинить)

**Reply by Monya00** [reply]
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/a960bc58-b98b-4f3e-97e6-38b389368183" />
я устала капец возиться с этим, помогите пожалуйста. все делаю правильно по инструкции, комп перезагружаю, сами домены вставлены правильно (фото прикреплено), но все равно твиттер выдает ошибку о том типа инета нет, а на сам сайт вообще не пропускает. ошибок в консоли где-то 15, не могу понять что конкретно не дает войти в твиттер

**Reply by ivaqis** [reply]
А сам запрет рабочий? Попробуй поставить какую-нибудь другую версию или использовать другой .bat конфиг
У меня стоит версия батника отсюда https://github.com/Flowseal/zapret-discord-youtube/discussions/3279, Твиттер работает нормально

**Reply by Monya00** [reply]
появилась новая проблема. теперь заместо ошибок с доменом он мне выдает эту хуйню. я скоро заплачу я искренне не поняла в чем проблема и че это за ошибка такая
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/0a81bdc0-1cdf-4783-a4c0-5f7d910113f5" />

**Reply by sadfov** [reply]
вставь такие домены:
ton.local.twitter.com
x.com
api.x.com
twitter.com
api.twitter.com
api.tweetdeck.com
abs.twimg.com
pbs.twimg.com
video.twimg.com
t.co
mobile.twitter.com
support.twitter.com
syndication.twitter.com
abs-0.twimg.com
upload.twitter.com
static.ads-twitter.com
и начинай перебирать alt'ы. у меня например на одном работает только ютуб, на другом только x. И нашёлся такой где работает и тот и тот. нужно просто перебирать, а ошибки эт база

**Reply by Slon-izMXA** [reply]
спасибо, досих пор работают домены
<img width="1913" height="867" alt="image" src="https://github.com/user-attachments/assets/5f5019a8-7766-464a-ba6f-4fdcc74a1ef6" />

**Comment by WickedSick91** [comment]
а кто знает какие домены у threads? тоже хотелось бы в паре с твиттером запускать

**Comment by shamil-m02-byte** [comment]
где можно найти такой же список адресов инстаграм?

**Reply by ivaqis** [reply]
По идее чет такое:
instagram.com
static.cdninstagram.com
facebook.com
static.xx.fbcdn.net
scontent.xx.fbcdn.net
scontent.cdninstagram.com
scontent-mxp2-1.cdninstagram.com
scontent-arn2-1.cdninstagram.com
scontent-arn2-1.xx.fbcdn.net
external-arn2-1.xx.fbcdn.net
threads.com

**Reply by shamil-m02-byte** [reply]
к сожалению в инстаграм и твитер не заходит с этим методом.все general попробовал

**Reply by ivaqis** [reply]
Вроде для инсты вообще надо в файле hosts на винде прописать эти значения, по обсуждениям поищи мб есть новее список для hosts:
157.240.9.174 instagram.com [www.instagram.com](http://www.instagram.com/)
52.201.119.26 instagram.com
52.201.106.156 instagram.com
52.200.181.107 instagram.com
34.234.52.85 instagram.com
103.234.123.224 instagram.fsub1-1.fna.fbcdn.net
49.44.90.32 instagram.fagr1-1.fna.fbcdn.net
154.115.242.161 instagram.fhga1-1.fna.fbcdn.net
31.13.71.52 graph.instagram.com
31.13.71.52 help.instagram.com
31.13.71.52 edge-chat.instagram.com
31.13.71.52 l.instagram.com
186.102.189.33 instagram.fbga1-4.fna.fbcdn.net
70.186.10.96 instagram.flas1-1.fna.fbcdn.net
94.97.236.33 instagram.fjed4-1.fna.fbcdn.net
179.60.192.36 facebook.com
157.240.2.35 facebook.com
179.60.192.36 [www.facebook.com](http://www.facebook.com/)
157.240.2.35 [www.facebook.com](http://www.facebook.com/)
179.60.192.36 fb.com
157.240.20.35 fb.com
157.240.194.18 mbasic.facebook.com
157.240.194.37 free.facebook.com
157.240.194.41 web.facebook.com
31.13.71.36 m.facebook.com
31.13.72.36 m.facebook.com
31.13.71.36 fbcdn.net
31.13.72.36 fbcdn.net
31.13.71.7 static.xx.fbcdn.net
157.240.194.27 static.xx.fbcdn.net
103.234.123.209 scontent.fsub1-1.fna.fbcdn.net
103.234.123.209 external.fsub1-1.fna.fbcdn.net
31.13.71.7 scontent.xx.fbcdn.net
157.240.194.27 scontent.xx.fbcdn.net
157.240.26.9 fburl.com
31.13.71.19 our.intern.facebook.com
157.240.194.9 our.intern.facebook.com
31.13.71.19 phabricator.intern.facebook.com
157.240.194.9 phabricator.intern.facebook.com
31.13.71.7 connect.facebook.net
157.240.194.27 connect.facebook.net
31.13.71.1 m.me
31.13.72.8 m.me
31.13.71.1 messenger.com
31.13.72.8 messenger.com
31.13.71.1 edge-chat.facebook.com
31.13.72.8 edge-chat.facebook.com
31.13.71.1 0-edge-chat.facebook.com
31.13.72.8 0-edge-chat.facebook.com
31.13.71.1 1-edge-chat.facebook.com
31.13.72.8 1-edge-chat.facebook.com
31.13.71.1 2-edge-chat.facebook.com
31.13.72.8 2-edge-chat.facebook.com
31.13.71.1 3-edge-chat.facebook.com
31.13.72.8 3-edge-chat.facebook.com
31.13.71.1 4-edge-chat.facebook.com
31.13.72.8 4-edge-chat.facebook.com
31.13.71.1 5-edge-chat.facebook.com
31.13.72.8 5-edge-chat.facebook.com
31.13.71.1 6-edge-chat.facebook.com
31.13.72.8 6-edge-chat.facebook.com
31.13.71.1 upload.facebook.com
31.13.72.8 upload.facebook.com
31.13.71.1 graph.facebook.com
31.13.72.8 graph.facebook.com
31.13.71.1 analytics.facebook.com
31.13.72.8 analytics.facebook.com
103.234.123.210 video.fsub1-1.fna.fbcdn.net
31.13.71.37 pixel.facebook.com
157.240.194.37 pixel.facebook.com
31.13.71.1 error.facebook.com
31.13.71.17 secure.facebook.com
31.13.71.7 connect.facebook.net

Еще как варик попробуй поставить DNS из списка от Adguard какой-нибудь в браузере

А вообще зайди в инструменты разработчика -> Network и смотри там все домены у которых статус Pending, там могут быть домены cdn серваков специфичных для твоего региона

И еще вот домены инсты:achat-followers-instagram.com
acheter-followers-instagram.com
acheterdesfollowersinstagram.com
acheterfollowersinstagram.com
bookstagram.com
carstagram.com
cdninstagram.com
chickstagram.com
ig.me
igcdn.com
igsonar.com
igtv.com
imstagram.com
imtagram.com
instaadder.com
instachecker.com
instafallow.com
instafollower.com
instagainer.com
instagda.com
instagify.com
instagmania.com
instagor.com
instagram-brand.com
instagram-engineering.com
instagram-help.com
instagram-press.com
instagram-press.net
instagram.com
instagramci.com
instagramcn.com
instagramdi.com
instagramhashtags.net
instagramhilecim.com
instagramhilesi.org
instagramium.com
instagramizlenme.com
instagramkusu.com
instagramlogin.com
instagramm.com
instagramn.com
instagrampartners.com
instagramphoto.com
instagramq.com
instagramsepeti.com
instagramtakipcisatinal.net
instagramtakiphilesi.com
instagramtips.com
instagramtr.com
instagran.com
instagranm.com
instagrem.com
instagrm.com
instagtram.com
instagy.com
instamgram.com
instangram.com
instanttelegram.com
instaplayer.net
instastyle.tv
instgram.com
intagram.com
intagrm.com
intgram.com
kingstagram.com
lnstagram-help.com
oninstagram.com
online-instagram.com
onlineinstagram.com
theinstagramhack.com
web-instagram.net
wwwinstagram.com
fbcdn.net
facebook.com
fbsbx.com
fb.me
facebook.me
static.xx.fbcdn.net
[www.facebook.com](http://www.facebook.com/)
scontent-hel3-1.xx.fbcdn.net
scontent.xx.fbcdn.net

**Reply by shamil-m02-byte** [reply]
спасибо

**Comment by FuiKvantis** [comment]
Подскажите, а на сайты grok.com и chatgpt.com действие zapret распространяется или только vpn поможет? В general добавил, но не открывается.

**Reply by ivaqis** [reply]
Не откроется, будет писать "This service is not available in your region.", тут либо впн, либо ставить dns сервер comss, т.к. эти сайты блокируются не dpi серверами, а самими их создателями по твоему гео

**Reply by FuiKvantis** [reply]
> Не откроется, будет писать "This service is not available in your region.", тут либо впн, либо ставить dns сервер comss, т.к. эти сайты блокируются не dpi серверами, а самими их создателями по твоему гео

как настроить dns сервер comss? я нашел эту инструкцию, но у меня в Opera GX даже такого пункта нет, как "Использовать безопасный DNS-сервер"
<img width="761" height="819" alt="image" src="https://github.com/user-attachments/assets/27e0f870-6471-4745-a733-b50b4645115e" />

**Reply by ivaqis** [reply]
> > Не откроется, будет писать "This service is not available in your region.", тут либо впн, либо ставить dns сервер comss, т.к. эти сайты блокируются не dpi серверами, а самими их создателями по твоему гео
> 
> как настроить dns сервер comss? я нашел эту инструкцию, но у меня в Opera GX даже такого пункта нет, как "Использовать безопасный DNS-сервер" <img alt="image" width="761" height="819" src="https://private-user-images.githubusercontent.com/189540984/534062026-27e0f870-6471-4745-a733-b50b4645115e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjgwMzY3MjYsIm5iZiI6MTc2ODAzNjQyNiwicGF0aCI6Ii8xODk1NDA5ODQvNTM0MDYyMDI2LTI3ZTBmODcwLTY0NzEtNDc0NS1hNzMzLWI1MGI0NjQ1MTE1ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDExMFQwOTEzNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMjExNmQ5NzVjMTlmOTk0ZmMwNDMzMmZlMDJjNDc4NzliOTdmMDY2OTM2MDc5MWVkZjM5ZGRkNzEzN2FjZWYxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wZISEvb_fIa3yZtnvapuhrq38YWDc5T21AxIN5h5gYo">

Скачай просто другой браузер для использования его с DNS сервером с обходом гео блока, чтобы случайно не попасть на госуслуги с ним, а то вдруг заблочат. Я для этого использую Cent Browser

**Comment by homyakovlexa-png** [comment]
Привет, народ, все сделал, как было описано выше, в твиттер заходит, но сразу получаю ошибку
<img width="634" height="234" alt="image" src="https://github.com/user-attachments/assets/cd647e78-1f4a-45a8-bdea-2cbb3947ce45" />
Что можно еще сделать?
<img width="247" height="455" alt="image" src="https://github.com/user-attachments/assets/61d9fbf6-dd9e-4b16-a741-39aa77f354c3" />

**Reply by homyakovlexa-png** [reply]
![Uploading image.png…]()

**Reply by ivaqis** [reply]
нажми f12, зайди в сеть/network и покажи какие запросы не проходят, т.е. те у которых pending или failed

**Reply by homyakovlexa-png** [reply]
<img width="663" height="810" alt="image" src="https://github.com/user-attachments/assets/a8593789-b04b-4e80-b786-28727b443a53" />

**Reply by homyakovlexa-png** [reply]
<img width="1920" height="753" alt="image" src="https://github.com/user-attachments/assets/cc06c171-e858-4a6c-9dce-61fab66f44bd" />
теперь в принципе перестал работать

**Reply by Tiluma23** [reply]
да не консоль, а network посмотри и обновите f5

**Reply by Tiluma23** [reply]
```
x.com
api.x.com
abs.twimg.com
play.google.com
video.twimg.com
twimg.com
pbs.twimg.com
twitter.com
analytics.twitter.com
com.cdn.cloudflare.net
```

**Reply by ivaqis** [reply]
> <img alt="image" width="663" height="810" src="https://private-user-images.githubusercontent.com/254669268/546236176-a8593789-b04b-4e80-b786-28727b443a53.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzAzOTcwODcsIm5iZiI6MTc3MDM5Njc4NywicGF0aCI6Ii8yNTQ2NjkyNjgvNTQ2MjM2MTc2LWE4NTkzNzg5LWIwNGItNGU4MC1iNzg2LTI4NzI3YjQ0M2E1My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMjA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDIwNlQxNjUzMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YmE5ODE5MDQ0ZjIxYzNiMThjOWQ5Yzg4ZmI0ZjhlNWU0YWRhZjhlNTM2MDk3MjdiZjVkZmEwMjBjN2Q1ZTFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.4C-smi15Cg6S5HSoNS12GOLfvicnCJp7CgDdEWjgOVk">

обнови страничку на этом экране

**Comment by Stalker-iodine** [comment]
<img width="983" height="193" alt="Снимок экрана 2026-02-09 174351" src="https://github.com/user-attachments/assets/f5fb7660-a3e4-436f-9db2-05b06a853229" />
Сейчас столкнулся с проблемой, что у меня Твиттер перестал работать (перестали грузиться посты и профили других пользователей). В качестве ссылок у меня было добавлено это:
<img width="204" height="313" alt="Снимок экрана 2026-02-09 174647" src="https://github.com/user-attachments/assets/b316c362-26f0-4880-9418-fca22d5435bc" />

**Reply by Stalker-iodine** [reply]
Также я проверил через f12 сеть и консоль и вот такие запросы у меня не проходят:
<img width="475" height="69" alt="Снимок экрана 2026-02-09 175651" src="https://github.com/user-attachments/assets/918444f2-1ef2-4f5e-914c-8c9d5980e5ed" />
<img width="529" height="85" alt="Снимок экрана 2026-02-09 175718" src="https://github.com/user-attachments/assets/4cc64275-2057-4fba-a4f4-d316024641c5" />
<img width="517" height="487" alt="Снимок экрана 2026-02-09 175734" src="https://github.com/user-attachments/assets/648b5fcd-07ae-4f3e-8d35-f26c89df3fab" />
<img width="547" height="784" alt="Снимок экрана 2026-02-09 175756" src="https://github.com/user-attachments/assets/60e761c5-9fb2-4343-9cbb-686e727d559f" />

**Reply by Tesra44** [reply]
Удалось победить проблему?

**Reply by Tesra44** [reply]
> twimg.com в генерал лист

Присутствовали его вариации, но добавил и его. К сожалению, картина не поменялась.
<img width="206" height="288" alt="изображение" src="https://github.com/user-attachments/assets/5da4c093-cb6b-450c-81af-59fdde0cad39" />

**Comment by windowssteam47905second-oss** [comment]
<img width="352" height="379" alt="image" src="https://github.com/user-attachments/assets/71743422-db60-42c7-88af-0537d050fc5c" />
<img width="590" height="270" alt="image" src="https://github.com/user-attachments/assets/d831c432-f3e4-4421-ab35-7cc78673942a" />

в list-general есть api.x.com, но в консоли появляется Error 404 (Bad Request)

**Comment by xNoHerox** [comment]
api.x.com
twitter.com
api.twitter.com
api.tweetdeck.com
abs.twimg.com
pbs.twimg.com
video.twimg.com
t.co
mobile.twitter.com
support.twitter.com
syndication.twitter.com
abs-0.twimg.com
upload.twitter.com
static.ads-twitter.com


вот это вставил в лист генерал, включил первый альт (без цифры который) заработало все, если не работает именно лента, пробуйте другие альты
